### ISSUE TITLE
Broken load-theme-buffer-local

### ISSUE THREAD

**Issue by sbp** [vic/color-theme-buffer-local#5]
Using load-theme-buffer-local 20120702.1336 from MELPA in emacs 24.3.1:

```lisp
(load-theme-buffer-local 'theme-name)
```

Breaks with:

```
Symbol's function definition is void: symbol
```

**Comment by vt5491** [comment]
This can be closed now.  It's simply not happening anymore.  It originally only happened only on windows.  Anyway, I'm running emacs 25.0 on windows and linux and this is not happening.  Not that going to emacs 25 made a difference -- it was working with emacs 24.3 and 24.5.  

I think it's safe to say this was an environmental issue and not a problem with 'load-theme-buffer-local'.  Whatever the environmental issue was it's now gone away.

**Comment by vic** [comment]
ok closing now.

**Comment by radrow** [comment]
The issue seems to be back on emacs 28.2. I am getting exactly the same error message. I am on Fedora.

**Comment by leonchas17** [comment]
I was also seeing this issue on Emacs for Mac OSX 28.2, and on Aquamacs. I was able to correct it by following the advice by @vt5491 on https://github.com/vic/color-theme-buffer-local/pull/6 (under the "files changed" tab) : after installing noflet, I changed the two lines mentioned on pull request #6 on my load-theme-buffer-local.el and byte-compiled it, then restarted Emacs again.  That got me the correct behavior. I'm still working on getting the correct functionality on Aquamacs, though.
